src/firefly/js/visualize/draw/DrawUtil.js

```diff
@@ -116,14 +116,15 @@ function drawText(drawTextAry,text, x,y,color,
         color,
         left:x,
         top:y,
+        //padding:'1px 3px 1px 3px',
         fontFamily,
         'fontSize': size,
         fontWeight,
         fontStyle,
-        'backgroundColor': 'white',
-        'MozBorderRadius': '5px',
-        'borderRadius': '5px',
-        'WebkitBorderRadius': '5px'
+        //'backgroundColor': 'white',
+        'MozBorderRadius': '1px',
+        'borderRadius': '1px',
+        'WebkitBorderRadius': '1px'
```

Comment:
I would still like to support background at the DrawUtil level.  I think you should add a extra parameter at the end (`backgroundColor`, default it to `null`  or `undefined`).  When a background color is not passed (the normal case), don't set a background color and don't set the padding.